### PR TITLE
Solution 1-4

### PR DESCRIPTION
**Pull request by Andrii256**
[DEMO LINK](https://andrii256.github.io/react_movies-list-add-form/)

### CODE REVIEW

**Review by Mikle25 — APPROVED** [review]

**Comment by irenhh** [comment]
image
empty movie (check for empty spaces)

File: src/components/NewMovie/NewMovie.js [hunk 1]
```diff
@@ -1,13 +1,70 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 
-export class NewMovie extends Component {
+export class NewMovie extends React.PureComponent {
   state = {};
 
+  Add = (event) => {
```
Comment (irenhh) [inline src/components/NewMovie/NewMovie.js:7]: why uppercase?

File: src/components/NewMovie/NewMovie.js [hunk 2]
```diff
@@ -1,13 +1,70 @@
...
+
+    this.props.onAdd({
+      title: title.value,
+      description: description.value,
+      imgUrl: imgUrl.value,
+      imdbUrl: imdbUrl.value,
+      imdbId: imdbId.value,
+    });
+
+    event.target.reset();
```
Comment (irenhh) [inline src/components/NewMovie/NewMovie.js:20]: don't interact with DOM directly

**Review by irenhh — CHANGES_REQUESTED** [review]

File: src/components/NewMovie/NewMovie.js (hunk 1 above)
Comment (irenhh) [reply src/components/NewMovie/NewMovie.js:13]: ?

**Review by irenhh — COMMENTED** [review]
image
I can still add empty title

**Comment by Andrii256** [comment]
validation is unnecessary task

**Review by irenhh — APPROVED** [review]

**Comment by irenhh** [comment]
it's a super basic validation

**Pull request closed without merging**